src/main/resources/assets/admin/common/js/ui/dialog/ModalDialog.ts

```diff
@@ -208,6 +208,12 @@ module api.ui.dialog {
             this.listOfClickIgnoredElements.push(elem);
         }
 
+        removeClickIgnoredEl(elem: api.dom.Element) {
+            if(this.listOfClickIgnoredElements.indexOf(elem) >= 0) {
+                delete this.listOfClickIgnoredElements.splice(this.listOfClickIgnoredElements.indexOf(elem), 1);
+            }
+        }
```

Comment:
Assign ``this.listOfClickIgnoredElements.indexOf(elem)`` to a ``const`` and use it the second time inside the `ìf``.